packages/next/src/client/components/layout-router.tsx

```diff
@@ -622,3 +622,6 @@ export default function OuterLayoutRouter({
     </>
   )
 }
+
+// TEST:
+// e2e/app-dir/layout-params
```

Comment:
Could this be a file in a directory instead? It seems pretty intensive having to list this in every file that should trigger the related test instead of say a folder that any file under it should trigger certain tests